lib/config.js

```diff
@@ -8,6 +8,14 @@ require('bedrock-validation');
 
 const cfg = config['authn-token-http'] = {};
 
+cfg.cookies = {};
+// client id to be baked into cookies that is used in client registration
+cfg.cookies.clientId = {
+  name: 'cid',
+  // 30 days expiration
+  maxAge: 30 * 24 * 60 * 60 * 1000
```

Comment:
We should nest the cookie options under `options` so that they can be configured more easily by applications. Some applications might need this cookie to be `SameSite=None` for example.

```suggestion
  options: {
    // 30 days expiration
    maxAge: 30 * 24 * 60 * 60 * 1000,
    httpOnly: true
  }
```